cla_backend/libs/eligibility_calculator/cfe_civil/income.py

```diff
@@ -0,0 +1,21 @@
+def translate_income(income, deductions):
+    value = {
+            "income": {
+                "receiving_only_statutory_sick_or_maternity_pay": False,
+                "frequency": "monthly",
+                "gross": income.earnings,
+                "benefits_in_kind": 0,
+                "tax": deductions.income_tax,
+                "national_insurance": deductions.national_insurance,
+                "prisoner_levy": 0,
+                "student_debt_repayment": 0
+            }
+    }
+    if income.self_employed:
+        return {
+            "self_employment_details": [ value ]
```

Comment:
"receiving_only_statutory_sick_or_maternity_pay" is for employment-only, so will get rejected by CFE, I believe
<img width="627" alt="Screenshot 2023-12-05 at 11 34 53" src="https://github.com/ministryofjustice/cla_backend/assets/307612/94417631-1eb5-4e71-976f-3312d0a4c15b">

Reply:
have added a test that proved it - had to remove benefits_in_kind as well